### ISSUE TITLE
Can you share the training curve for RoBERT?

### ISSUE THREAD

**Issue by gaopengcuhk** [pytorch/fairseq#951]
Is it possible to share the training curve of RoBERT with x coordinate time while y coordinate the cross entropy loss. That will benefit my debug training.

**Comment by gaopengcuhk** [comment]
In RoBERTa, you report ppl. I would like to confirm that the following code in the masked_lm means the ppl in RoBERTa. If not, please clarify this.


def get_average_masked_score(self, model, src_tokens, mask, **net_input):
        """Mask a set of tokens and return their average score."""
        masked_tokens = src_tokens.clone()
        masked_tokens[mask.byte()] = self.mask_idx
        net_output = model(src_tokens=masked_tokens, **net_input, last_state_only=True)
        lprobs = F.log_softmax(net_output[0], dim=-1, dtype=torch.float32)
        lprobs = lprobs.gather(-1, src_tokens.unsqueeze(-1)).squeeze(-1)
        mask = mask.type_as(lprobs)
        score = (lprobs * mask).sum(dim=-1) / mask.sum(dim=-1)
        return score

**Comment by myleott** [comment]
That code is actually for the WNLI GLUE task (more details coming soon). The right place to look is here: https://github.com/pytorch/fairseq/blob/master/fairseq/criterions/masked_lm.py. Ppl in the paper is just the exponentiated loss.

I will get back to you re: learning curve. We may not have exact timestamps, but I can send you loss vs. num. steps.

**Comment by ngoyal2707** [comment]
![Screen Shot 2019-09-19 at 12 35 49 PM](https://user-images.githubusercontent.com/7836935/65264235-32b69e80-dadc-11e9-9ccd-8c01d902b3d4.png)

**Comment by Eric-Wallace** [comment]
Hi @ngoyal2707 just to clarify, this curve is the train perplexity of RoBERTa-Large for the best run?
